modules/drivers/numpy/wscript

```diff
@@ -34,10 +34,9 @@ def configure(conf):
                        includes=inc_dir,
                        libpath=lib_dir,
                        msg='Checking for library numpy' + warningmessage, 
-                       mandatory=False) 
-            
+                       mandatory=True)
```

Comment:
It's not mandatory True vs. False so much as that you have to check for a different header right?